lib/commons/dom/is-focusable.js

```diff
@@ -9,22 +9,33 @@ import { getNodeFromTree } from '../../core/utils';
  * @memberof axe.commons.dom
  * @instance
  * @param {HTMLElement} el The HTMLElement
+ * @param {object} options Optional. If set to {programatically: false}, the function will return true
  * @return {Boolean} The element's focusability status
  */
-export default function isFocusable(el) {
+export default function isFocusable(el, options = { programmatically: false }) {
   const vNode = el instanceof AbstractVirtualNode ? el : getNodeFromTree(el);
 
   if (vNode.props.nodeType !== 1) {
     return false;
   }
 
+  var tabindex = vNode.attr('tabindex');
+
+  if (
+    tabindex &&
+    options.programmatically === true &&
+    parseInt(tabindex, 10) === -1
```

Comment:
`-1` is not the only valid negative tabindex value

```suggestion
    parseInt(tabindex, 10) < 0
```

Reply:
You'll also want to modify one of the tests for another negative value other than `-1`